extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -47,8 +51,7 @@ export class AmpAnim extends AMP.BaseElement {
   buildCallback() {
     this.img_ = new Image();
     this.img_.setAttribute('decoding', 'async');
-    this.propagateAttributes(['alt', 'aria-label',
-      'aria-describedby', 'aria-labelledby'], this.img_);
+    this.propagateAttributes(ATTRIBUTES_TO_PROPAGATE, this.img_);
```

Comment:
Propagating the `src`/`srcset` here is potentially expensive if the it points to a gif or something animated.

Reply:
Gotcha